content/javascript/concepts/storage/terms/getItem/getItem.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '.getItem(key)'
+Description: 'Returns the value of the key on the given data item.'
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags: 
+  - 'APIs'
+  - 'Data'
+  - 'Methods'
+  - 'Storage'
+  - 'Values'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.getItem()`** method takes a key as an argument and returns the associated value. If the key is not present in the `Storage` object, it returns `null`.
+
+The `Storage` Object, which can be either a `localStorage` object or a `sessionStorage` object.
+
+## Syntax
+
+```pseudo
+localStorage.getItem(keyName)
+
+sessionStorage.getItem(keyName)
+```
+
+The `keyName` parameter is required and it's a string containing the name of the key you want to retrieve the value of.
+
+> **Note:** The `Storage.length` property can be used to test whether the storage object is empty or not.
+
+## Example
+
+Get the value of the specified local storage item:
+
+```codebyte/javascript
```

Comment:
There are still changes missing here. The block is still set as a codebyte and the code should be revised to be a more complete example that could be conceivably used as is, something like:
localStorage.setItem("Seuss", "Green Eggs and Ham")
console.log(localStorage.getItem("Seuss")

And the output should be in a separate ```shell block